### PR TITLE
Update production

### PR DESCRIPTION
**Pull request by Robinlovelace**
To all, especially @nikolai-b, there are a few minor bug fixes in here. Please review and merge when ready. Wider question: are there manual fixes we'll have to do each time we do a merge from test -> prod. Imagine it will be something we'll need to a few times over the coming months.

### CODE REVIEW

File: regions_www/m/server.R [hunk 1]
```diff
@@ -333,9 +333,10 @@ shinyServer(function(input, output, session) {
    }
   }, priority = 3)
 
-  # Only requred to run if the region changes
+  # Only requred to run if the region changes (as that affects purpose) or the purpose changes (as that affects geographies)
   observe({
     region$current
+    input$purpose
```
Comment (nikolai-b) [inline regions_www/m/server.R]: @AnnaGoodman1 I don't believe this should be needed

`region$purposes_present` and `region$geographies_present` only change when the `region$current` changes

https://github.com/npct/pct-shiny/blob/3374d9bb0b4a6b707431c19606a845ea48309aa6/regions_www/m/server.R#L284-L286

Would you mind explaining what this fixes?

Comment (Robinlovelace) [reply regions_www/m/server.R]: This (before it did not switch back to MSOAs):

peek 2017-09-13 07-39

Comment (AnnaGoodman1) [reply regions_www/m/server.R]: @nikolai-b : fixes this issue: https://github.com/npct/pct-shiny/issues/620 (sorry, should have cross-referenced)

Comment (nikolai-b) [reply regions_www/m/server.R]: Right, that makes sense, thanks

**Comment by Robinlovelace** [comment]
@nikolai-b planning to merge this by midday.

**Pull request merged**